### DISCUSSION TITLE
animals

### DISCUSSION THREAD

**Discussion by vincent-d-izo** [blakeblackshear/frigate#964] [Ask A Question]
hello, 
cars, personn work very well, i try to get animal worked, but didn't succeed.
did anyone have make it work ?
thanks

**Comment by savikko** [comment]
Yes. Detects at least cats.

Try to estimate how many pixels i.e. cat will fill and then adjust settings accordingly.

**Reply by vincent-d-izo** [reply]
Hi, this for your answer, 
Did you add cat or animal as category?

**Reply by savikko** [reply]
Cat.

**Comment by vincent-d-izo** [comment]
anybody succeed with bird ?
